guides/doc-Project_API/topics/proc_applying-errata-to-a-host-or-host-collection.adoc

```diff
@@ -0,0 +1,49 @@
+[id="applying-errata-to-a-host-or-host-collection"]
+= Applying errata to a host or host collection
+
+You can use the API to apply errata to a host, host group, or host collection.
```

Comment:
>     * Should we add the API procedure to the "normal"/existing procedure? Aka. is it a good idea to split off procedure by means to interacting with Foreman?

That's a good point. I can imagine that a more thorough review of how to actually merge the new API modules with the existing guides will be required (what I can't imagine is who would have the capacity for it in the foreseeable future :upside_down_face:). For example, some time ago, I used the API guide's overview module and added it (with a few edits) to the Overview guide: https://docs.theforeman.org/nightly/Planning_for_Project/index-katello.html#API-Overview_planning  This module will now effectively be duplicated but it works well within the Overview guide. There may be more modules like that that would fit well in the existing guides.

Adding the API procedures from the API guide to appear alongside the CLI and web UI procedures (and also Ansible procedures) seems like a good idea. It would take a lot of effort, though.